packages/nodes-base/nodes/Orbit/ActivityDescription.ts

```diff
@@ -0,0 +1,237 @@
+            },
+        ],
+        default: 'create',
+        description: 'The operation to perform.',
+    },
+] as INodeProperties[];
+
+export const activityFields = [
+
+/* -------------------------------------------------------------------------- */
+/*                                activity:create                             */
+/* -------------------------------------------------------------------------- */
+    {
+        displayName: 'Workspace',
+        name: 'workspaceId',
+        type: 'options',
+        typeOptions: {
+            loadOptionsMethod: 'getWorkspaces',
+        },
+        default: '',
+        required: true,
+        displayOptions: {
+            show: {
+                resource: [
+                    'activity',
+                ],
+                operation: [
+                    'create',
+                ],
+            },
+        },
+        description: 'The workspace',
+    },
+    {
+        displayName: 'Member ID',
+        name: 'memberId',
+        type: 'string',
+        default: '',
+        required: true,
+        displayOptions: {
+            show: {
+                resource: [
+                    'activity',
+                ],
+                operation: [
+                    'create',
+                ],
+            },
+        },
+        description: 'Member ID',
+    },
+    {
+        displayName: 'Title',
+        name: 'title',
+        type: 'string',
+        default: '',
+        required: true,
+        displayOptions: {
+            show: {
+                resource: [
+                    'activity',
+                ],
+                operation: [
+                    'create',
+                ],
+            },
+        },
+        description: 'Title',
+    },
+    {
+        displayName: 'Additional Fields',
+        name: 'additionalFields',
+        type: 'collection',
+        placeholder: 'Add Field',
+        displayOptions: {
+            show: {
+                resource: [
+                    'activity',
+                ],
+                operation: [
+                    'create',
+                ],
+            },
+        },
+        default: {},
+        options: [
+            {
+                displayName: 'Description',
+                name: 'description',
+                type: 'string',
+                default: '',
+                description: 'A description of the activity; displayed in the timeline',
+            },
+            {
+                displayName: 'Link',
+                name: 'link',
+                type: 'string',
+                default: '',
+                description: 'A URL for the activity; displayed in the timeline',
+            },
+            {
+                displayName: 'Link Text',
+                name: 'linkText',
+                type: 'string',
+                default: '',
+                description: 'The text for the timeline link',
+            },
+            {
```

Comment:
We can remove the score field, this is something we are removing in the API soon.